packages/core-v2/src/semantic-model/concepts/concepts-utils.ts

```diff
@@ -14,4 +20,47 @@ export function isSemanticModelRelationship(resource: Entity | null): resource i
 export const SEMANTIC_MODEL_GENERALIZATION = "generalization"; // todo use proper IRI
 export function isSemanticModelGeneralization(resource: Entity | null): resource is SemanticModelGeneralization {
     return resource?.type.includes(SEMANTIC_MODEL_GENERALIZATION) ?? false;
-}
\ No newline at end of file
+}
+
+export function isSemanticModelAttribute(resource: Entity | null): resource is SemanticModelRelationship {
+    if (!isSemanticModelRelationship(resource)) {
+        return false;
+    }
+    return isSemanticModelRelationPrimitive(resource);
+}
+
+export function isSemanticModelRelationPrimitive(resource: SemanticModelRelationship) {
+    const isConceptAPrimitiveType = (concept: string | null) => {
+        return concept == null || concept == "" || resource.id.endsWith("#attribute"); // TODO: temporary workaround for SKOS #449
```

Comment:
Yes. In the future, we will add here also the list of IRIs that are considered as attributes. Such as xsd:string or xsd:decimal.